hivemq-edge/src/main/java/com/hivemq/edge/modules/adapters/ProtocolAdapterConstants.java

```diff
@@ -20,6 +20,20 @@
  */
 public interface ProtocolAdapterConstants {
 
+    enum TAG {
+        UDP,
+        TCP,
+        SERIAL,
+        ETHERNET
+    }
+
+    enum CATEGORY {
+        INDUSTRIAL,
```

Comment:
Ah there :-)

Reply:
Our internal uses enums to strongly type